pkg/arm/arm.go

```diff
@@ -29,9 +29,9 @@ func New(ctx context.Context, log *logrus.Entry, cs *api.OpenShiftManagedCluster
 	switch cs.Config.PluginVersion {
 	case "v3.2":
 		return v3.New(ctx, log, cs, testConfig), nil
-	case "v4.2", "v4.3":
+	case "v4.2", "v4.3", "v4.4":
 		return v4.New(ctx, log, cs, testConfig), nil
-	case "v5.0":
+	case "v5.1":
```

Comment:
Should this not be additive like the "4.2", "4.3", etc. case or is no one running "5.0" so it doesn't matter?

Reply:
5.0 is not yet out so we dont care